src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 SignatureV4.md

```diff
@@ -5,6 +5,8 @@ description: 'SignatureV4 is used to sign or pre-sign requests to AWS services u
 excerpt: 'SignatureV4 is used to sign and pre-sign requests to AWS services using the Signature V4 algorithm'
 ---
 
+<BlockingAwsBlockquote />
```

Comment:
Does it make sense to have this quote here if the example doesn't include any updates to use async/await? (maybe a silly question 🫥)

Reply:
Oh wait, we're removing this quote from the services that now have async/await. That makes sense. 😅

Reply:
I've left it here because although it doesn't use async await (yet), it involves some pretty costly cryptographic operations. I would leave it as is in the context of this specific PR, and would reiterate once either the signature code is async, or we rephrase this block completely 👍🏻

Reply:
That makes sense, thanks for the explanation 🙇 .